extensions/wikia/AdEngine/js/spec/AdTracker.spec.js

```diff
@@ -253,7 +253,7 @@ describe('ext.wikia.adEngine.adTracker', function () {
 
 		timer = adTracker.measureTime('test/event', {abc: 'def', xyz: 123});
 
-		windowMock.wgNow = new Date(new Date().getTime() - 999);
+		windowMock.wgNow = new Date(windowMock.wgNow.getTime() - 110);
```

Comment:
Oh, OK. We discussed it offline. And in the assertions below we can find `110` and `888` as well, so it seems the author came up with the numbers. Let's leave it as it is.